.github/workflows/nodejs.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - run: npm install
-      - run: npm run build -- --prod
+      - run: npm run build
```

Comment:
I don't know how this option came about. The `prod` option was not defined in package.json, so I deleted it.